mkat_fpga_tests/test_cbf.py

```diff
@@ -692,12 +733,11 @@ def test_sensor_values(self):
 
         # Check all sensors statuses if they are nominal
         for sensor in correlator_fixture.rct.sensor.values():
-            LOGGER.info(sensor.name + ':'+ str(sensor.get_value()))
+            LOGGER.info(sensor.name + ' :'+ str(sensor.get_value()))
             Aqf.equals(sensor.get_status(), 'nominal',
                 'Sensor status fail: {}, {} '
                     .format(sensor.name, sensor.get_status()))
 
-
```

Comment:
@brickZA Reverted code back to original state, and made separate branch for changes.